src/event-notion/index.ts

```diff
@@ -324,91 +325,138 @@ export const pingForTAPandCSIDeadlines = async (
   databaseId: string,
   config: EventNotionPipelineConfig,
 ) => {
-  // The day for which we ping for any non-submitted events.
-  // This is just basically to look for events that are 23 days away from now.
-  //
   // Ideally, do ALL network queries in one and filter later.
-  const firstDayToPingForTODO = DateTime.now().plus({ days: 23 });
-  const secondDayToPingForTODO = DateTime.now().plus({ days: 22 });
-  const thirdDayToPingForTODO = DateTime.now().plus({ days: 21 });
-  const firstDayToPingForInProgress = DateTime.now().plus({ days: 16 });
-  const secondDayToPingForInProgress = DateTime.now().plus({ days: 15 });
-  const thirdDayToPingForInProgress = DateTime.now().plus({ days: 14 });
+  var dateTimeNow = DateTime.now();
+
+  // Master list of all reminders in the following format:
+  // title — Top of embed, bolded
+  // colour — colour of embed, since GREEN was used for key pings/succesful imports, and RED is semantically errors
+  // dates — further itemized by number of days in advance a reminder ping should be sent:
+  //    pingroles — names of roles to be pinged, found in settings
+  //    date — exact date in the future you would wish to receive pings for today
+  //    prop — category, corresponds to a series of statuses on the Notion existing as "<prop> Status"
+  //    propStatus — the status of the above prop category
+  const daysToPing = [
+    {
+      'title': 'AS Funding Deadline',
+      'colour': 'BLUE',
+      'dates': {
+        56: {
+          'pingRoles': ['logisticsTeamID'],
+          'date': dateTimeNow.plus({ days: 56}),
```

Comment:
There's probably a cleaner way to write this instead of having to put 56 in two places. Maybe writing a constructor function?

```tsx
const createReminder = (days, title, color, ...) => {}
```